css/editor.css

```
@@ -237,6 +236,43 @@ body {
   font-style: italic;
   color: #777;
 }
+
+/* Styles for obsolete HPO Terms and tooltips */
+#panogram .disorder.obsolete {
+  position: relative;
+  display: inline-block;
+}
+
+#panogram .disorder.obsolete::before {
+  font-family: FontAwesome;
+  font-size: 12px;
+  content: "\f071";
+  color: #ff8a00;
+  position: absolute;
+  left: -15px;
```

Tiny suggested change to align it a little nicer (I don't like magic numbers though so feel free to leave it as is 😁 )

```suggestion
  left: -18px;
  top: 3px;
```